src/snowflake/connector/auth.py

```diff
@@ -463,6 +472,25 @@ def read_temporary_credentials(self, host, user, session_parameters):
                 host, user, MFA_TOKEN
             )
 
+    @staticmethod
+    def get_token_from_private_key(user, account, privatekey_path, key_password):
```

Comment:
If this function doesn't use the class itself please don't make it a static function.
In Python not everything has to be in a class and if you wanted to use it from the outside it'd still be `snowflake.connector.auth.get_token_from_pricate_key`
Also add type hints please!

Reply:
Removed- now it's just at the bottom of the class but not using the class object